neurobench/examples/mswc_fscil/mswc_fscil_proto.py

```diff
@@ -1,59 +1,60 @@
-#
-# NOTE: This task is still under development.
-#
-
-import sys
-sys.path.append("/home3/p306982/Simulations/fscil/algorithms_benchmarks/")
 import os
+import json
+
+import numpy as np
 
 import torch
+import torch.nn as nn
 import torch.nn.functional as F
-import numpy as np
 import copy
 from tqdm import tqdm
 
-from torch.utils.data import DataLoader, ConcatDataset
-import torchaudio.transforms as T
+import wandb
 
-# from torch_mate.data.utils import IncrementalFewShot
+from torch.utils.data import DataLoader, ConcatDataset
 
 from neurobench.datasets import MSWC
 from neurobench.datasets.MSWC_IncrementalLoader import IncrementalFewShot
 from neurobench.examples.mswc_fscil.M5 import M5
 from neurobench.models import TorchModel
 
-from neurobench.examples.model_data.sparchSNNs import SNN
-from neurobench.examples.model_data.sparchSNNs import RadLIFLayer
+from neurobench.examples.mswc_fscil.sparchSNNs import SNN
+from neurobench.examples.mswc_fscil.sparchSNNs import RadLIFLayer
 
 from neurobench.benchmarks import Benchmark
-from neurobench.preprocessing import MFCCProcessor
-from neurobench.preprocessing import S2SProcessor
-
-from cl_utils import *
+from neurobench.preprocessing import MFCCProcessor, S2SProcessor
 
+from weight_consolidation_utils import examples_per_class
```

Comment:
Not needed actually

Reply:
Resolved by #b7dfcfa4be8984550be14da395c6a068933587b9.